sdk/src/main/java/com/mapbox/maps/renderer/egl/EGLCore.kt

```diff
@@ -176,16 +192,60 @@ internal class EGLCore(
     return EGL10.EGL_SUCCESS
   }
 
-  /**
-   * Checks for EGL errors.
-   */
-  private fun checkEglErrorNoException(msg: String): Boolean {
+  internal fun addRendererStateListener(listener: RendererSetupErrorListener) {
```

Comment:
Isn't it supposed to be launched on main thread already?

Reply:
adding listener - yes (however does not matter tbh), why are you asking?

Reply:
Since there's `mainHandler.post {` that is unneeded then